homeassistant/components/glances/__init__.py

```diff
@@ -62,6 +62,11 @@ def host(self):
         """Return client host."""
         return self.config_entry.data[CONF_HOST]
 
+    @property
+    def name(self):
+        """Return client name."""
+        return self.config_entry.data[CONF_NAME]
+
```

Comment:
```suggestion
```
Not needed as you can just use name in DeviceInfo as both of them goes back to entry.data[CONF_NAME]

Reply:
See my reply to another comment